packages/react/src/components/TableHandles/hooks/useTableHandlesPosition.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  BlockNoteEditor,
+  BlockSchema,
+  DefaultBlockSchema,
+  DefaultInlineContentSchema,
+  DefaultStyleSchema,
+  InlineContentSchema,
+  StyleSchema,
+} from "@blocknote/core";
+import { UiComponentPosition } from "../../../components-shared/UiComponentTypes";
+import { useEffect, useRef, useState } from "react";
+import { offset, useFloating, useTransitionStyles } from "@floating-ui/react";
+
+export function useTableHandlesPosition<
+  BSchema extends BlockSchema = DefaultBlockSchema,
+  I extends InlineContentSchema = DefaultInlineContentSchema,
+  S extends StyleSchema = DefaultStyleSchema
+>(
+  editor: BlockNoteEditor<BSchema, I, S>
+): {
+  rowHandle: UiComponentPosition;
+  colHandle: UiComponentPosition;
+} {
+  const [show, setShow] = useState<boolean>(false);
+
+  const [draggedCellOrientation, setDraggedCellOrientation] = useState<
+    "row" | "col" | undefined
+  >(undefined);
+  const [mousePos, setMousePos] = useState<number | undefined>();
+
+  const [, setForceUpdate] = useState<number>(0);
+
+  const referencePosCell = useRef<DOMRect>();
+  const referencePosTable = useRef<DOMRect>();
+
+  const rowFloating = useFloating({
+    open: show,
+    placement: "left",
+    middleware: [offset(-10)],
+  });
+  const colFloating = useFloating({
+    open: show,
+    placement: "top",
+    middleware: [offset(-12)],
+  });
+
+  const rowTransitionStyles = useTransitionStyles(rowFloating.context);
+  const colTransitionStyles = useTransitionStyles(colFloating.context);
+
+  useEffect(() => {
+    return editor.tableHandles!.onPositionUpdate((position) => {
+      // console.log("update", state.draggingState);
+      setShow(position.show);
+      setForceUpdate(Math.random());
```

Comment:
why do we need this here? maybe needs a comment (might have been my code ofc)

Reply:
I think it's something you added, after removing it everything still seems to work just fine so no clue what it's for. Any ideas?